align_system/configs/adm/pipeline.yaml

```diff
@@ -0,0 +1,52 @@
+name: pipeline
+
+defaults:
+  # Import defaults into this namspace (adm) as @name, for further
+  # customization
+
+  # Shared variables / components
+  - /attribute@mj: moral_judgment
+  - /inference_engine@structured_inference_engine: outlines_structured_inference_engine
+  - /template/scenario_description@scenario_description_template: with_relevant_char_info
+  - /template/prompt@prompt_template: comparative_regression
+  - /template/output_schema@comparative_regression_choice_schema: comparative_regression_choice
+  # ADM components to be used in "steps"
+  - /adm_component/misc@step_definitions.format_choices: itm_format_choices
+  - /adm_component/icl@step_definitions.regression_icl: regression
+  - /adm_component/regression@step_definitions.comparative_regression: comparative
+  - /adm_component/alignment@step_definitions.scalar_alignment: avg_dist_scalar
+  - /adm_component/misc@step_definitions.justification_from_reasonings: justification_from_reasonings
```

Comment:
@jadie1 note the additional step here to get the justification for the selected action